CHANGELOG.md

```diff
@@ -7,11 +7,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## 1.3.0 - Jan 30, 2020
+
+This release adds support for the new rippled protocol buffers and adds browser compatibility support. 
+
+### New Protocol Buffers from rippled
+
+rippled implemented protocol buffer support natively, which uses a new and incompatible set of protocol buffers. The implementation in rippled was completed in: ripple/rippled#3159. Xpring-JS is now capable of using these protocol buffers. This change is opt-in and non-breaking. 
+
+To switch to these protocol buffers, pass `true` to the `useNewProtocolBuffer` parameter in `XpringClient`'s constructor. The default for this field is false. 
+
+
+### Browser Compatibility
+
+This library is now compatible with web browsers. Internally, the library makes different network requests when in the browser. Users can force these request types in the node environment by passing `true` for the `forceWeb` parameter in `XpringClient`'s constructor. This boolean parameter is intended for compatibility testing only. This change is opt-in and non-breaking.
```

Comment:
Maybe note that we plan on removing the `forceWeb` parameter at some point in the future. (Once we have a way to test the web stuff using Puppeteer, and not hacking Node).